vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -566,6 +567,194 @@ int64_t WindowsTextInputComponentView::sendMessage(uint32_t msg, uint64_t wParam
   return Super::sendMessage(msg, wParam, lParam);
 }
 
+WPARAM PointerPointToPointerWParam(const winrt::Microsoft::ReactNative::Composition::Input::PointerPoint &pp) noexcept {
```

Comment:
Do we want this helper somewhere where it can be used, rather than dedicated to WindowsTextInputComponentView?

Reply:
Hopefully we're not converting back to window messages regularly.  -- If it something we need other places we can extract it then.